src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.Publish.targets

```diff
@@ -143,9 +143,11 @@ Copyright (c) .NET Foundation. All rights reserved.
     </ItemGroup>
 
     <!--
-        Not using SkipUnchangedFiles="true" because the application may want to change
-        one of these files and not have an incremental build replace it.
-        -->
+      SkipUnchangedFiles is not used on the Copy task because it will detect size differences between the
+      source and destination files.  Users may have modified the published files and we don't want to copy
+      the source files again just because the published files differ by size.  Therefore, this target uses
```

Comment:
I think it's more interesting to say that PreserveNewest means to not revert to source copy if any changes have been made since publish.

Reply:
Actually, the next sentence covers that so OK.

Reply:
I'll amend.

Reply:
How about:

```xml
<!-- 
    PreserveNewest means that we will only copy the source to the destination if the source is newer.
    SkipUnchangedFiles is not used for that purpose because it will copy if the source and destination
    differ by size.  Instead, this target uses inputs and outputs to only copy when the source is newer.
    -->
```